WalletWasabi/Services/WasabiSynchronizer.cs

```diff
@@ -163,7 +157,8 @@ public void Start(TimeSpan requestInterval, TimeSpan feeQueryRequestInterval, in
 									return;
 								}
 
-								response = await WasabiClient.GetSynchronizeAsync(hashChain.TipHash, maxFiltersToSyncAtInitialization, estimateMode, Cancel.Token)
+								using WasabiClient wasabiClient = WasabiClientFactory.NewBackendClient();
```

Comment:
> AFAIK understand we were using the same Tor identity for every sync request, now we are getting a new one every time. Is this correct?

I was not sure what behavior was intended here. It can be easily fixed though.

Reply:
@nopara73 Could you possibly say what you find to be correct solution here?

Reply:
> Are we not exhausting Tor's circuit pool with this? @lontivero what do you think about this?

Short answer: theoretically no. What we do here is using a feature called _stream isolation_ that creates a new _stream_ (tcp connection) which will be attached to one of the available (if there is one) circuits. Isolated streams are "isolated" because they don't share circuit with other streams and then we have that one-to-one map: one stream, one circuit.

Long answer: probably yes. Circuits are expensive resources because their creation involves a lot of roundtrips and negotiations and that's why Tor keeps a list of circuits. Forcing isolation means that we need more circuits than what we would need if multiple streams were able to share the same circuit. OTOH these connections have a short live.

**We should NOT use isolated streams here.**

Reply:
@molnard I have modified the PR according to Lucas's suggestion.